desktop/apps/artwork/client/index.coffee

```diff
@@ -72,9 +73,20 @@ module.exports =
           require('../../../components/current_auctions/index.jsx').default unless context.is_closed
           require '../components/artist_artworks/index.coffee' if context.is_closed
           require '../components/related_artworks/index.coffee' if context.is_closed
-          require '../components/related_artists/index.coffee'
+          require '../components/related_artists/index.coffee',
         ]
 
+      # Make some Auctions-only layout tweaks. See https://github.com/artsy/auctions/issues/360#event-1017559326
+      # for more info.
+      postInit: ->
+        $currentAuctions = $ '.artwork-current-auctions'
+        $relatedRail = $ '.artwork-artist-related-rail'
+        paddingTop = if $currentAuctions.length then '42px' else '31px'
+
+        $relatedRail
+          .css('border-top', 0)
+          .css('padding-top', paddingTop)
```

Comment:
Sometimes, it can be worth providing feedback to design on what things don't have easy solutions with CSS. It could be preferable in some cases to go with an alternative design that lets the component be more self-contained than to end up with something we might have to revisit when edge cases and weird interactions pop up.